packages/rn-tester/js/examples/TextInput/TextInputExample.ios.js

```diff
@@ -767,6 +767,27 @@ exports.examples = ([
       );
     },
   },
+  {
+    title: 'Multiline colored highlight for text input',
+    render: function (): React.Node {
+      return (
+        <View>
+          <TextInput
+            style={styles.multiline}
+            selectionColor={'green'}
+            defaultValue="Highlight me"
+            multiline={true}
+          />
+          <TextInput
+            style={styles.multiline}
+            selectionColor={'rgba(86, 76, 205, 1)'}
+            defaultValue="Highlight me"
+            multiline={true}
+          />
+        </View>
+      );
+    },
+  },
```

Comment:
Could we just add this to the existing `Colored highlight for text input` and/or then upstream that change?